### ISSUE TITLE
Okay serious issue now

### ISSUE THREAD

**Issue by pranavr2003** [ryanfortner/exagear-rpi#5]
I followed the tutorial exactly. Now, for some reason, `exagear` isn't running due to a licensing issue, `sudo` isn't working system-wide, `ng` an other stuff aren't working at all. All of a sudden I'm not able to use `sudo` with anything, `npm` and others have gone, `neofetch` is not found, etc. 

```
pranav@raspberrypi:/etc $ cd sudoers
bash: cd: sudoers: Not a directory
pranav@raspberrypi:/etc $ nano sudoers
pranav@raspberrypi:/etc $ sudo nano sudoers
>>> /etc/sudoers: syntax error near line 27 <<<
sudo: parse error in /etc/sudoers near line 27
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin

pranav@raspberrypi:~/exagear $ sudo
>>> /etc/sudoers: syntax error near line 27 <<<
sudo: parse error in /etc/sudoers near line 27
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin

pranav@raspberrypi:~/Desktop/ts_ange/demo1 $ npm
bash: npm: command not found

pranav@raspberrypi:~ $ neofetch
bash: neofetch: command not found
```

...and of course, exagear isn't running - `bash: exagear: command not found`

Do be my saviour, @ryanfortner !!!

**Comment by pranavr2003** [comment]
For some reason, the licensing wasn't working with `curl ...` that's there in the installation instructions in the README. So I found it was an already closed issue here, and tried `$ wget https://archive.org/download/exagear-desktop_202111/patch.sh; sudo bash patch.sh`, as you had suggested in #2 . This issue began since running that one.

**Comment by pranavr2003** [comment]
Closed by accident.

**Comment by pranavr2003** [comment]
Okay I just restarted my system, and there's a surprise which is waiting for me: 

```
pranav@raspberrypi:~ $ sudo
usage: sudo -h | -K | -k | -V
usage: sudo -v [-AknS] [-g group] [-h host] [-p prompt] [-u user]
usage: sudo -l [-AknS] [-g group] [-h host] [-p prompt] [-U user] [-u user]
            [command]
usage: sudo [-AbEHknPS] [-r role] [-t type] [-C num] [-D directory] [-g group]
            [-h host] [-p prompt] [-R directory] [-T timeout] [-u user]
            [VAR=value] [-i|-s] [<command>]
usage: sudo -e [-AknS] [-r role] [-t type] [-C num] [-D directory] [-g group]
            [-h host] [-p prompt] [-R directory] [-T timeout] [-u user] file ...
pranav@raspberrypi:~ $ exagear
Starting /bin/bash  in the guest image /opt/exagear/images/debian-9
bash: ng: command not found
bash: /home/pranav/.local/bin/thefuck: /usr/bin/python3: bad interpreter: No such file or directory
pranav@raspberrypi:~ $ ng
bash: ng: command not found
pranav@raspberrypi:~ $ exagear
bash: exagear: command not found
pranav@raspberrypi:~ $ sudo
>>> /etc/sudoers: syntax error near line 27 <<<
sudo: parse error in /etc/sudoers near line 27
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin
```

!!!
Is `ng`/Angular the offender?! Why does it pop an issue with `ng` when running `exagear` the first time? What does it have to do with it? I'm so confused right now. I've definitely not edited any of the scripts, and my environment was working just fine before all this. 

PS
If this isn't magic, I don't know what is

**Comment by pranavr2003** [comment]
IT IS SOLVED NOW WITH #4 AND @frankhval 's reply. 

For anyone looking for a solution and have the same issue as mine, head over to #4 . Nothing's actually happened to your system, and your dev environments are all safe. It does seem like 'By some strange magic ExaGear session was looking into the main system's sudoers'.
